.env

```
@@ -101,6 +101,7 @@ _APP_FUNCTIONS_MEMORY_SWAP=256
 _APP_FUNCTIONS_RUNTIMES=node-16.0,php-8.0,python-3.9,ruby-3.0,java-16.0
 _APP_EXECUTOR_SECRET=your-secret-key
 _APP_EXECUTOR_HOST=http://exc1/v1
+_APP_BROWSER_HOST=http://appwrite-browser:3000/v1
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The _APP_BROWSER_HOST key should go before the _APP_COMPRESSION_ENABLED key

(UnorderedKey)